CleverTap/clevertap-ios-sdk
# SDK-2946 GitHubActions

Set up CI/CD workflows in github actions:

Mini workflows to:
- Build SDK Project
- Run XCTests
- Check PR Title format
- Check PR description format
- Check if mandatory files are updated(Changelog, podspec, BuildInfo)

Main workflows for:
- PR from task branch to develop
- PR from develop to master
